bullet_train/app/helpers/account/dates_helper.rb

```diff
@@ -1,17 +1,54 @@
 module Account::DatesHelper
-  def display_date(timestamp, format: :default, date_format: nil)
+  def display_date(timestamp, custom_date_format = nil, format: :default, date_format: nil)
+    return nil unless timestamp
     format = date_format if date_format
-    localize(local_time(timestamp).to_date, format: format) if timestamp
+
+    if format && format == :default
```

Comment:
And this is an attempt to preserve the old behavior if someone doesn't explicitly include a `format` or `date_format`. I suspect that this might interfere with the idea that people could use localization strings to customize things, but I'm not entirely sure how all of those pieces fit together.